go/tools/bazel_testing/bazel_testing.go

```diff
@@ -179,7 +179,7 @@ func TestMain(m *testing.M, args Args) {
 // hide that this code is executing inside a bazel test.
 func BazelCmd(args ...string) *exec.Cmd {
 	cmd := exec.Command("bazel")
-	cmd.Args = append(cmd.Args, "--nosystem_rc", "--nohome_rc")
+	cmd.Args = append(cmd.Args, "--nohome_rc")
```

Comment:
Could you add a comment explaining why `--nosystem_rc` isn't used here? This is a good example for why we don't want it, even though it appears to be a must for integration tests otherwise.

Reply:
Done!